sql_generators/glean_usage/glean_app_ping_views.py

```diff
@@ -162,7 +162,8 @@ def generate_per_app(
                     full_view_id,
                     "view.sql",
                     rendered_view,
-                    skip_existing=True,
+                    skip_existing=get_table_dir(output_dir, full_view_id) / "view.sql"
+                    in self.override_skip(),
```

Comment:
- Same suggestion about calling `self.override_skip()` within a loop.
- This formatting makes it kind of confusing to read, so I'd suggest a more readable formatting (if Black allows it):
```suggestion
                    skip_existing=(
                        get_table_dir(output_dir, full_view_id) / "view.sql"
                        in self.override_skip()
                    ),
```